website/docs/components/time/partials/code/component-api.md

```diff
@@ -0,0 +1,60 @@
+## Component API
+
+The Time component makes use of the Hds-time service, which provides various time and formatting related functions as well as a tracked `Now` property representing current time in milliseconds, which continuously updates.
+
+It uses two helper functions, the format-date helper, which formats a date according to the passed in options, and the format-relative helper, which returns a relative string according to the passed in value and units.
```

Comment:
[question] Should this be a semicolon instead of a comma?

```suggestion
It uses two helper functions; the format-date helper, which formats a date according to the passed in options, and the format-relative helper, which returns a relative string according to the passed in value and units.
```

Reply:
A comma is ok as I have it or I could change to use a colon which would also be correct. (semi-colon doesn't work here.)

I will edit to change to a colon.